site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/controller/SiteController.java

```diff
@@ -54,4 +55,9 @@ public SiteController(SiteService siteService) {
         return siteService.findParentSiteIds(id);
     }
 
+    @GetMapping("/assigned")
```

Comment:
Unauthorised endpoint with a body or params to retrieve site name and id - no need for auth as auth is carried out when requesting site sensitive data when retrieving all sites or an individual site with an id.

Reply:
Thanks, helpful to have this comment to understand why no authz